project/Settings.scala

```diff
@@ -5,7 +5,7 @@ import sbtassembly.AssemblyKeys._
 object Settings {
   val buildName = "hazelcast-spark"
   val buildVersion = "0.1-SNAPSHOT"
-  val buildScalaVersion = "2.10.5"
+  val buildScalaVersion = "2.11.8"
```

Comment:
@eminn I moved to Scala 2.10.6 but I don't know how to clearly remove this [commit](https://github.com/VEINHORN/hazelcast-spark/commit/9ee17ede2875e0d77132490d08ea96c52a5b458e) Maybe you have some ideas about it.